docs/layers/data/data.mdx

```diff
@@ -1,7 +1,47 @@
 ---
+title: Provision Databases
 sidebar_class_name: hidden
 ---
 
-# Data
+import Intro from '@site/src/components/Intro';
+import PrimaryCTA from '@site/src/components/PrimaryCTA';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-TODO
+<Intro>
+  There are many options for provisioning databases in AWS. The reference architecture provides support for a variety of databases, including Aurora PostgreSQL, Aurora MySQL, DynamoDB, RDS, and more.
+
+  Each database has its own unique features and benefits, so you can choose the one that best fits your needs or deploy multiple databases to support different use cases.
+</Intro>
+
+<Tabs queryString="database">
+  <TabItem value="aurora-postgres" label="Aurora PostgreSQL">
```

Comment:
We may want to break this apart to SQL and NoSQL databases, as we support more than we have width of the screen.

Reply:
separated and added documentdb to fill out nosql options